SharedLibs/ThreeAddons/src/examples/jsm/loaders/GLTFLoader.js

```diff
@@ -1621,8 +1621,6 @@ class GLTFMeshGpuInstancing {
 				// Just in case
 				Object3D.prototype.copy.call( instancedMesh, mesh );
 
-				// https://github.com/mrdoob/three.js/issues/18334
-				instancedMesh.frustumCulled = false;
```

Comment:
Upgrade from Thee.js 1.151.0 to 1.152.0
This is potentially a big performance improvement.

Reply:
Do we know why they have been able to activate this and not before?

Reply:
From what I read it has something to do with AABB evaluation. As deformation is calculated by GPU directly, the AABB was using the geometry before the deformation and it messed the culling.